cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenSecureFields.js

```diff
@@ -38,6 +45,43 @@
         card.mount(cardNode);
     };
 
+    var oneClickValid;
+    function renderOneClickComponents() {
+        var componentContainers = document.getElementsByClassName("cvc-container");
+        try{
+            jQuery.each(componentContainers, function(i, oneClickCardNode){
+                var container = document.getElementById(oneClickCardNode.id);
+                oneClickCard = checkout.create('card', {
+                    //Get selected card, send in payment request
+                        originKey: originKey,
+                        loadingContext: loadingContext, // The environment where we should loads the secured fields from
+                        // Specific for oneClick cards
+                        details: [{"key":"cardDetails.cvc","type":"cvc"}], // <--- Pass the specific details for this paymentMethod
+                        oneClick: true, //<--- enable oneClick 'mode'
+                        storedDetails: {
+                            "card": {
+                                "expiryMonth": "",
```

Comment:
shouldn't these card data come from payment instruments (via /listRecurring or /paymentmethods)?

Reply:
No, we will use default SFRA functionality to list stored payment methods. 
Then we will populate CVC secured fields per stored method.